homeassistant/components/onewire/onewireentity.py

```diff
@@ -0,0 +1,57 @@
+"""Support for 1-Wire environment sensors."""
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class OneWireEntity(Entity):
+    """Implementation of a 1-wire Entity."""
```

Comment:
```suggestion
    """Implementation of a 1-Wire Entity."""
```

Reply:
Updated in 91674ed2f4a5d563c09230231656aeb29a074b38